hypothesis_bio/hypothesis_bio.py

```diff
@@ -106,3 +127,24 @@ def fasta(draw):
     """Generate strings representing sequences in FASTA format.
     """
     return draw(parsed_fasta())["fasta"]
+
+
+@composite
+def fastq(draw, fasta_source=fasta(), quality_source=fastq_quality()):
+    """Generate strings representing sequences in FASTQ format.
+    """
+    fasta_sequence = draw(fasta_source)
+    raw_sequence = fasta_sequence["sequence"]
+    sequence_id = fasta_sequence["comment"]
+    sequence_length = len(raw_sequence)
+    quality_string = draw(quality_source(sequence_length))
+    return (
+        "@"
+        + sequence_id
+        + "\n"
+        + raw_sequence
```

Comment:
You need a newline after the `raw_sequence`

Reply:
Ahh ofc. Fixed it now